app/client/components/pages/creditTransferListPage.jsx

```diff
@@ -29,7 +29,7 @@ const mapDispatchToProps = (dispatch) => {
 
 class CreditTransferListPage extends React.Component {
   componentDidMount() {
-      this.props.loadCreditTransferList({per_page:50, page:1})
+      this.props.loadCreditTransferList()
```

Comment:
load whole credit transfer list - if this takes to long @enjeyw, we could load first 1000, then async load the rest (showing a loading bar top left or something).

Reply:
redux is built to update pretty seamlessly

Reply:
what's wrong with paginating this right now?

Reply:
because then search doesn't work